modules/mesh-task/variables.tf

```diff
@@ -100,3 +100,21 @@ variable "gossip_key_secret_arn" {
   type        = string
   default     = ""
 }
+
+variable "acls" {
```

Comment:
should this be `acls_enabled`?

Reply:
Right now this is mirroring `tls` var for consistency. I like that they are consistent but don't have strong preference for which one is better. Should I change both to `{tls,acls}_enabled`?